src/rsa/rsa.rs

```diff
@@ -43,6 +46,7 @@ fn parse_public_key(input: untrusted::Input)
 
 struct PositiveInteger {
     value: Option<*mut BIGNUM>,
+    bit_len: usize,
```

Comment:
This, plus the logic in `from_be_bytes` to initialise the field, is the only non-trivial change I've made in the rebase.